### PR TITLE
Fix: Bottom Navbar

### PR DESCRIPTION
**Pull request by Bahugunajii**
### Description
- Fixed isActive issue for nested path segments

### Checklist
- [x] Self-testing

### CODE REVIEW

File: components/BottomNavigationBar.tsx [hunk 1]
```diff
@@ -11,9 +11,14 @@ const BottomNavigationBar = () => {
   const pathname = usePathname();
 
   const isActive = (path: string) => {
-    return pathname === path;
+    if (path === '/') {
```
Comment (pritamps) [inline components/BottomNavigationBar.tsx:14]: Sorry, don't understand -- why was this needed?

**Review by pritamps — APPROVED** [review]

File: components/BottomNavigationBar.tsx (hunk 1 above)
Comment (Bahugunajii) [reply components/BottomNavigationBar.tsx:14]: Just to make bottom navbar icons to be filled with color if they are active (even for nested routes).
So for content and class library, out route looks like: (/library/content?additionalParams). Therefore, we want the navbar icons remain activated (or filled with primary color) for these additional path segments.
